docker/Dockerfile

```diff
@@ -1,9 +1,10 @@
 # Global ARGs shared by all stages
+ARG ARCH=
```

Comment:
I think this needs to be more descriptive e.g `IMAGE_PREFIX_ARCH`

Reply:
Actually this isn't really needed as we use buildkit and a base image which already has support for the target platforms. Thus I have removed the `ARCH` prefix in the latest series. Damn musle memory from times before multi platform support in Docker :-)